infra/modules/worklytics-connectors/outputs.tf

```diff
@@ -21,4 +21,4 @@ output "todos" {
 
 output "next_todo_step" {
   value = max(values(module.source_token_external_todo)[*].next_todo_step...)
```

Comment:
this needed `try()` to, so I just ported both to https://github.com/Worklytics/psoxy/pull/432